src/config.js

```diff
@@ -61,7 +71,18 @@ class Config {
       fetchMaxAttempts !== undefined && fetchMaxAttempts !== null
         ? fetchMaxAttempts
         : 4
+    this.throttleConfig = {
+      throttleRequests: throttleRequests || false,
+      throttleLimit: throttleLimit || 3,
+      throttleInterval: throttleInterval || 125,
+      throttleStrict: throttleStrict || false,
+      httpKeepAlive: httpKeepAlive || false,
+      httpKeepAliveInterval: httpKeepAliveInterval || 10000
```

Comment:
Same as above if we are removing http keep alive from this PR then this isn't needed

Reply:
I'll update it, thank you